packages/main/src/Dialog.js

```diff
@@ -212,10 +215,43 @@ class Dialog extends Popup {
 	}
 
 	onExitDOM() {
-		this._dragMouseMoveHandler = null;
-		this._dragMouseUpHandler = null;
+		delete this._screenResizeHandler;
+		delete this._dragMouseMoveHandler;
+		delete this._dragMouseUpHandler;
+		delete this._resizeMouseMoveHandler;
+		delete this._resizeMouseUpHandler;
+	}
+
+	show() {
+		ResizeHandler.register(this, this._screenResizeHandler);
+		ResizeHandler.register(document.body, this._screenResizeHandler);
+
+		this.style.visibility = "hidden";
```

Comment:
This doesn't seem to do anything. Why is it needed?

Reply:
Removed it